src/assets/styles/missions.css

```diff
@@ -0,0 +1,88 @@
+table {
+  width: 90vw;
+  margin: 15px auto;
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+}
+
+tr:nth-child(even) {
+  background-color: #ddd;
+}
+
+td,
+th {
+  border: 1px solid #c0c0c0;
+  text-align: left;
+  padding: 15px;
+}
+
+.mission_name {
+  vertical-align: top;
+  width: 10vw;
+  font-weight: bold;
+  font-size: 1.2rem;
+  justify-content: flex-start;
+}
+
+.reserve_btn {
+  background-color: transparent;
+  font-size: 1rem;
+  padding: 10px;
+  color: black;
+  border: 1px solid black;
+}
+
+.cancel_btn {
+  background-color: transparent;
+  font-size: 1rem;
+  padding: 10px;
+  border: 1px solid red;
+  color: red;
+}
+
+.cancel_btn:hover {
+  background-color: transparent;
+  font-size: 1rem;
+  padding: 10px;
+  border: 1px solid #c0c0c0;
+  color: red;
+}
+
+.reserve_btn:hover {
+  background-color: transparent;
+  font-size: 1rem;
+  padding: 10px;
+  color: black;
+  border: 1px solid #c0c0c0;
+}
```

Comment:
- [ ] Kindly follow the CSS best practices and combine all the selectors that share the same rules. 😄 🙏 This is for cleaner and more optimized code. 💯 
_You can find about this tip and other useful ones right [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)._
>Kindly apply this request to all similar situations through your files. 🙏